adlfs/cli.py

```diff
@@ -305,10 +310,12 @@ def do_put(self, line):
         parser.add_argument('remote_path', type=str, nargs='?', default='.')
         parser.add_argument('-b', '--chunksize', type=int, default=2**28)
         parser.add_argument('-c', '--threads', type=int, default=None)
+        parser.add_argument('-f', '--force', action='store_true')
```

Comment:
No default needed. It automatically sets to false by default. `store_true` is the action taken when the `force` flag is present.